api/src/main/java/org/openmrs/module/addresshierarchy/config/AddressConfigurationLoader.java

```diff
@@ -48,11 +49,10 @@ public static String getChecksumsPath() {
 		return Paths.get(OpenmrsUtil.getApplicationDataDirectory(),
 				"configuration_checksums").toString();
 	}
-
-	public static void loadAddressConfiguration() {
-
-		final ConfigDirUtil configUtil = new ConfigDirUtil(getConfigPath(), getChecksumsPath(), "addresshierarchy");
-
+	
```

Comment:
I think we can eliminate passing in the "domain" here.  That seems redundant and unnecessary.